src/EnergyPlus/PlantComponentTemperatureSources.cc

```diff
@@ -87,257 +85,47 @@ namespace PlantComponentTemperatureSources {
     //  predefined (but variable) boundary temperature.
 
     // METHODOLOGY EMPLOYED:
-    // Called by plantloopequipment, model accepts inputs, and calculates a
+    // Called by PlantLoopEquipment, model accepts inputs, and calculates a
     // thermal response using new plant routines such as SetComponentFlowRate
 
-    // Using/Aliasing
-    using namespace DataPrecisionGlobals;
-    using namespace DataLoopNode;
-    using DataGlobals::DisplayExtraWarnings;
-    using DataHVACGlobals::SmallWaterVolFlow;
-    using DataPlant::TypeOf_WaterSource;
-    using General::RoundSigDigits;
-    using General::TrimSigDigits;
-
     // MODULE PARAMETER DEFINITIONS:
-    int const TempSpecType_Constant(-1);
-    int const TempSpecType_Schedule(-2);
+    int const modTempSpecType_Constant(-1);
+    int const modTempSpecType_Schedule(-2);
```

Comment:
In general, the constant integers at the top of modules are fine to leave as-is.  If anything, I'd suggest trying to change it to an enum class where possible (I know it's not always feasible).